subjects/java/projects/lets-travel/audit/README.md

```diff
@@ -0,0 +1,92 @@
+#### Comprehension
+
+##### Understanding of Integrated Technologies
```

Comment:
This was not addressed, the structure is the same. You have two options here:
1- You must create a "roleplay" so that the auditor ask the right questions to the student
2 - make the questions like this:
```
##### Ask the students to elaborate on how Elasticsearch contributes to the system's search and autocomplete features.
###### Was the student able to explain how Elasticsearch contributes to the system's search and autocomplete features?
```